prepare_check_cleanup/reconcile.py

```diff
@@ -192,6 +185,49 @@ def _check_url_result(
     return True
 
 
+def _get_tag_name() -> str:
+    """Get the name of the tag to use for the content.
+
+    Returns:
+        The name of the tag.
+
+    """
+    actions_yaml = Path("action.yaml").read_text(encoding="utf-8")
+    return yaml.safe_load(actions_yaml)["inputs"]["base_tag_name"]["default"]
+
+
+def _check_git_tag_exists(test_name: str, github_repo: Repository, should_exist: bool) -> bool:
```

Comment:
This method is called always with `should_exist`set to true. Can we remove that argument? Also, it seems a little obscure to me

Reply:
Thanks for noticing that, yes I had removed the use case for `should_exist=False` and not removed the argument, I'll remove that shortly

Reply:
The reason this check exists is because the new way to check for content conflicts is to look at the commit tagged `upload-charm-docs/base-content` (which can be customised) instead of a specific branch. This function checks that the tag has actually been set after the action is run, which is important for the action to work

Reply:
Unnecessary argument removed